### DISCUSSION TITLE
Remove database name in sql query

### DISCUSSION THREAD

**Discussion by marklai1998** [prisma/prisma#17369] [Q&A]
Background:
My company uses DAL to intercept SQL connection for load balancing etc. One of the limitations is that it cannot do cross-database queries. When creating such an attempt, the connection will get dropped.

Now all my SQL generated by prisma is automatically appended with the database name

```
SELECT `my_db`.`my_table`.`my_column` ....
```

And it got detected as a "cross database" query although its not.

I know it sounds stupid but is it possible to drop the DB name if I only have 1 connection? Thanks

**Comment by nurul3101** [comment]
Hello @marklai1998 👋 

I don't think it is possible at the moment. There isn't any configuration yet that would remove the database name from generated SQL queries.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
